webapp/src/components/configForms/TaskForm.tsx

```diff
@@ -28,6 +28,10 @@ const TaskForm: React.FC<TaskFormProps> = props => {
     return useObserver(() => {
         const { config: task, validation } = props
 
+        const onTimeoutInHours = (value: string) => {
+            task.timeoutInHours = value
```

Comment:
you should parse string to number here

Reply:
As I wrote above, I found precedences in front end only to deal with strings like this. So the change would be more global, thus as separate commit pelease.